rebalance/rebalance.py

```diff
@@ -418,7 +490,11 @@ def maybe_rebalance_pairs(plugin: Plugin, ch1, ch2, failed_channels: list):
         plugin.log(f"Try to rebalance: {scid1} -> {scid2}; amount={amount}; maxfee={maxfee}")
         start_ts = time.time()
         try:
-            res = rebalance(plugin, outgoing_scid=scid1, incoming_scid=scid2, msatoshi=amount, maxfeepercent=0, retry_for=1200, exemptfee=maxfee)
+            res = rebalance(plugin, outgoing_scid=scid1, incoming_scid=scid2,
+                    msatoshi=amount, retry_for=1200, maxfeepercent=0,
+                    exemptfee=maxfee, maxhops=6, msatfactor=1)
```

Comment:
Maybe you meant this?
```suggestion
            res = rebalance(plugin, outgoing_scid=scid1, incoming_scid=scid2,
                    msatoshi=amount, retry_for=1200, maxfeepercent=0,
                    exemptfee=maxfee, maxhops=plugin.maxhops, msatfactor=plugin.msatfactor)
```

Reply:
Actually this was by intention, to have the long running (non interactive) rebalances for `rebalanceall` using least aggressive values, while the manual (likely interative) commands using this optimization that tries to give faster results.

What values do we want for `rebalanceall`? certainly not plugin defaults of maxhops 5  and msatfactor 4.